packages/core/src/browser/tree/tree-selection-state.spec.ts

```diff
@@ -0,0 +1,256 @@
+/*
+ * Copyright (C) 2018 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { enableJSDOM } from '../../browser/test/jsdom';
```

Comment:
Is it necessary? the tree model should not have dom dependencies, at least it was an idea

Reply:
Yeah, I agree. I do not like that either. I verify it once more whether we need it or not...

Reply:
Yes, that works.